src/api.ts

```diff
@@ -173,6 +173,11 @@ export const serviceUsageOrigin = utils.envOverride(
   "FIREBASE_SERVICE_USAGE_URL",
   "https://serviceusage.googleapis.com"
 );
+// TODO: Change to correct URL later.
```

Comment:
Please assign an issue or a contributor to this TODO.

Reply:
removing this - internal details.